model/generate.go

```diff
@@ -274,8 +274,8 @@ func (g *GeneratedProject) saveNewBuildsAndTasks(ctx context.Context, v *Version
 		return errors.Errorf("project '%s' not found", p.Identifier)
 	}
 
-	activatedTasksInExistingBuilds, err := addNewTasks(ctx, activationInfo, v, p, newTVPairsForExistingVariants,
-		existingBuilds, syncAtEndOpts, projectRef.Identifier, g.TaskID)
+	activatedTasksInExistingBuilds, err := addNewTasks(ctx, activationInfo, v, p, projectRef, newTVPairsForExistingVariants,
```

Comment:
We can move the project ref lookup at L269 here into addNewTasks

Reply:
I think the project ref lookup is still necessary for [`addNewBuilds`](https://github.com/evergreen-ci/evergreen/pull/5635/files#diff-96655072263d498b6dc1571c45fc69e886abcc4bc9994343a80af2f54375c7bcR283).

Reply:
Whoops. Missed that